.github/workflows/ci-cd.yml

```
@@ -107,13 +107,343 @@
+    - name: Get deployment URL
+      env:
+        RAILWAY_TOKEN: ${{ secrets.RAILWAY_TOKEN }}
+      id: deployment-url
+      run: |
+        # Get the deployment URL
+        DEPLOY_URL=$(railway domain | head -n1 | awk '{print $1}')
+        if [ -z "$DEPLOY_URL" ]; then
+          echo "❌ Could not retrieve deployment URL"
+          exit 1
+        fi
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)